packages/clay-drop-down/src/DropDownWithDrilldown.tsx

```diff
@@ -82,7 +88,7 @@ export const ClayDropDownWithDrilldown: React.FunctionComponent<IProps> = ({
 	spritemap,
 	trigger,
 }: IProps) => {
-	const [active, setActive] = React.useState(false);
+	const [active, setActive] = React.useState(initiallyOpen);
```

Comment:
Hey @julien I think that just adding a new property to define the initial state will probably not support the desired use case which is to close DrillDown when clicking on the element, in which case we need to make DrillDown controlled by the developer, support two new properties, `active `and `onActiveChange` to allow the state to be controlled. We have a hook that handles this, to control the state internally when these two properties do not exist.

```js
const [internalActive, setInternalActive] = useInternalState({
	initialValue: false,
	onChange: onActiveChange,
	value: active,
});
```

Example https://github.com/liferay/clay/blob/master/packages/clay-drop-down/src/DropDownWithItems.tsx#L402-L406

Reply:
Hey @matuzalemsteles sure,
After talking with @veroglez, I thought we just needed the drop down to be initially open. I'll update.

Reply:
@matuzalemsteles looking at this 

```js
const [internalActive, setInternalActive] = useInternalState({
	initialValue: false,
	onChange: onActiveChange,
	value: active,
});
```

One thing I don't understand is why we would set `initialValue` to `false`, since we want to add the possibility for the menu to be open immediately.

Reply:
@julien in this case it is just to define the initial value, if we want to make it configurable we must also have a new property called `initialValue`.

Reply:
> After talking with @veroglez, I thought we just needed the drop down to be initially open. I'll update.

Hmm, it didn't seem very clear on the issue but either way, we can make it consistent with `DropDownWithItems`.

Reply:
:+1:

Reply:
T.I.L.:  [`useInternalState`](https://github.com/liferay/clay/blob/d639da89b4ac9eddb66a0b5270da046c4da6565d/packages/clay-shared/src/useInternalState.ts#L19-L35)